### PR TITLE
👩‍✈️ Fix TS error + add github actions

### PR DESCRIPTION
**Pull request by stefanoeb**
TS was complaining when using more than one child, so I fixed it and also added some GH actions to make our life easier

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ declare module 'seapig' {
   }
 
   export default function seapig(
-    children: ReactElement,
+    children: ReactElement[],
```
Comment (stefanoeb) [inline index.d.ts]: this is the TS fix

**Comment by nem035** [comment]
Thanks for the fix my man 💪 

Just one thing, I'm wondering why are we adding github actions cause CircleCI [(config)](https://github.com/enkidevs/seapig/blob/master/.circleci/config.yml) is doing the [same thing](https://github.com/enkidevs/seapig/blob/master/package.json#L16)? 😄

**Comment by stefanoeb** [comment]
Ah I see, it doesn't report anything on the checks tho. 

I'll remove it for now

**Comment by stefanoeb** [comment]
hecho ✅

**Comment by nem035** [comment]
Hmmmm good point, I'll look into why that's happening.

[The builds are running](https://circleci.com/gh/enkidevs/seapig) 🤷‍♂

**Comment by nem035** [comment]
Ok, I added the checks

**Comment by stefanoeb** [comment]
🍠

**Pull request merged**